scripts/display-travis-status.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const debug = require('debug')('display_travis_status')
+const pollTravis = require('../lib/pollTravis')
+
+module.exports = function (app) {
+  app.on('pull_request.opened', (event) => {
+    const owner = event.repository.owner.login
+    const repo = event.repository.name
+    if (repo !== 'nodejs.org') return
```

Comment:
I was leaning more against removing that check all together. It isn't there today, and we haven't had the need to add either.